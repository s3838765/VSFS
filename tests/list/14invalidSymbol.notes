NOTES V1.0
@validFile
=validDir/
#validIgnoreLine
%invalidSymbol
