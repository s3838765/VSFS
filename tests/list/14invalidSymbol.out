An unknown file type was found by the compiler (%). Terminating program.
